bazel/mini_chromium.BUILD

```diff
@@ -12,12 +12,165 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-licenses(["notice"])  # Apache v2.0
+licenses(["notice"])
 
 package(
     default_visibility = ["//visibility:public"],
 )
 
+DEFAULT_MINI_CHROMIUM_WIN_COPTS = [
```

Comment:
Would it make sense to pull some of the common copts into a common DEFAULT_BASE_COPTS? (i.e. any shared flags between DEFAULT_MINI_CHROMIUM_WIN_COPTS and DEFAULT_MINI_CHROMIUM_LINUX_COPTS)

Reply:
done; created DEFAULT_MINI_CHROMIUM_BASE_COPTS